ynh_systemd_action/ynh_systemd_action

```diff
@@ -28,32 +28,26 @@ ynh_systemd_action() {
     local log_path="${log_path:-/var/log/$service_name/$service_name.log}"
     local length=${length:-20}
     local timeout=${timeout:-300}
-    local wait_starting=true
 
-    if [[ -z "${line_match:-}" ]]
+    if [[ -n "${line_match:-}" ]]
     then
-        wait_starting=false
-    fi
-
-    ynh_clean_check_starting () {
-        # Stop the execution of tail.
-        kill -s 15 $pid_tail 2>&1
-        ynh_secure_remove "$templog" 2>&1
-    }
-
-    echo "Starting of $service_name" >&2
-    systemctl $action $service_name || ( journalctl --lines=$length -u $service_name >&2 && false)
```

Comment:
Why do you call `systemctl $action $service_name` only if `if [[ -n "${line_match:-}" ]]`. The idea is in all case to do the action for the service and if `line_match` is defined to wait until the service is completely started.

Reply:
Indeed, it doesn't work...
However, we have to start tailing the log before using the action on the service, or it could happens that the service is already started (or whatever else) when the log is analysed.
So, I think we should divide this part between the tail part, and the time out part.

Reply:
> So, I think we should divide this part between the tail part, and the time out part.

Yes, we probably need this. (It's probably why I created the `$wait_starting` variable)

Reply:
Done

Reply:
Thanks